.github/workflows/build-beta.yml

```
@@ -25,48 +30,25 @@ jobs:
           # Specifying `ref` ensures that the head commit is checked out directly.
           ref: ${{ env.HEAD_COMMIT_HASH }}
 
-      - name: Needs beta build
-        if: ${{ env.BRANCH != 'master' }}
-        id: needs-beta-build
-        run: |
-          version="${BRANCH/Version-v/}"
-          commit_message=$(git show -s --format=%s HEAD)
-          beta_version_regex="Version v[0-9]+\.[0-9]+\.[0-9]+-beta\.[0-9]+"
-
-          if [[ "$commit_message" =~ $beta_version_regex ]]; then
-              printf '%s\n' "Creating a build for $version with $commit_message"
-              echo "NEEDS_BETA_BUILD=true" >> "$GITHUB_OUTPUT"
-          else
-              printf '%s\n' 'Commit message does not match commit message for beta pattern; skipping beta build'
-              echo "NEEDS_BETA_BUILD=false" >> "$GITHUB_OUTPUT"
-          fi
-
       - name: Checkout and setup high risk environment
-        if: ${{ steps.needs-beta-build.outputs.NEEDS_BETA_BUILD == 'true' }}
         uses: MetaMask/action-checkout-and-setup@v1
         with:
           is-high-risk-environment: true
           ref: ${{ env.HEAD_COMMIT_HASH }}
           skip-allow-scripts: true
 
       - name: Run beta build
-        if: ${{ steps.needs-beta-build.outputs.NEEDS_BETA_BUILD == 'true' }}
         env:
-          INFURA_PROJECT_ID: ${{ vars.INFURA_PROJECT_ID }}
-          INFURA_BETA_PROJECT_ID: ${{ vars.INFURA_BETA_PROJECT_ID }}
-          SEGMENT_BETA_WRITE_KEY: 00000000000
-          SENTRY_DSN: https://fake@sentry.io/0000000
-          ENABLE_MV3: true
+          INFURA_PROJECT_ID: ${{ secrets.INFURA_PROJECT_ID }}
+          SENTRY_DSN: ${{ secrets.SENTRY_DSN_DEV }}
```

We would need to pass this in as `SENTRY_DSN_DEV` as well, otherwise it won't get used

```suggestion
          SENTRY_DSN_DEV: ${{ secrets.SENTRY_DSN_DEV }}
```